frontend/.gitignore

```diff
@@ -17,7 +17,7 @@ dist-ssr
 # Editor directories and files
 .vscode/*
 !.vscode/extensions.json
-.idea
+../.idea/
```

Comment:
creative but hard to read or for someone to find out why the folder is ignored. its best practice to have the gitignore in the directory in which you want to ignore files